sklearn/neighbors/tests/test_neighbors.py

```diff
@@ -658,6 +659,29 @@ def test_radius_neighbors_regressor(n_samples=40,
             y_pred = neigh.predict(X[:n_test_pts] + epsilon)
             assert_true(np.all(abs(y_pred - y_target) < radius / 2))
 
+    # test that nan is returned when no nearby observations
+    for weights in ['uniform', 'distance']:
+        neigh = neighbors.RadiusNeighborsRegressor(radius=radius,
+                                         weights=weights,
+                                         algorithm='auto')
+
+        neigh.fit(X, y)
+
+
+        X_test_nan = np.ones((1, n_features))*-1
+
+        empty_warning_msg = ("One or more samples have no neighbors "
+                             "within specified radius; predicting NaN.")
+
```

Comment:
please consider to remove some unnecessary blank lines both in the code and in the test.